resources/zib/zib-DrugUse.xml

```diff
@@ -0,0 +1,230 @@
+      <short value="StartDate" />
+      <definition value="The date on which the patient started using." />
+      <alias value="StartDatum" />
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.6" />
+        <comment value="StartDate" />
+      </mapping>
+    </element>
+    <element id="Observation.effective[x]:effectivePeriod.end">
+      <path value="Observation.effective[x].end" />
+      <short value="StopDate" />
+      <definition value="The date on which the patient quit using" />
+      <alias value="StopDatum" />
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.7" />
+        <comment value="StopDate" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueCodeableConcept">
+      <path value="Observation.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="DrugUseStatus" />
+      <definition value="Indication stating whether there was any drug use in the past or present." />
+      <alias value="DrugsGebruikStatus" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.4.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.3" />
+        <comment value="DrugUseStatus" />
+      </mapping>
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Relevant comments on the drug use." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.9" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:amount">
+      <path value="Observation.component" />
+      <sliceName value="amount" />
+    </element>
+    <element id="Observation.component:amount.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="363908000" />
```

Comment:
https://bits.nictiz.nl/browse/ZIB-1466 says 228390007

Reply:
Its says that 363908000 is a safer solution @jduwel

Reply:
No. See last comment on zib ticket

Reply:
Ok sorry